test/teiserver_web/tachyon/matchmaking_test.exs

```diff
@@ -88,6 +89,22 @@ defmodule Teiserver.Tachyon.MatchmakingTest do
     }
   end
 
+  defp add_startboxes(attrs, n_teams) do
```

Comment:
(nit) It seems to me that it would be valuable to make this a public function and add a single unit test demonstrating and testing the behavior. 

Not that I think it is super risky, but just to satisfy the curious that it was tested and can easily have tests added to it if edge cases are found.